skbeam/core/accumulators/projector.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python
+
+from __future__ import division
+import numpy as np
+
+"""
+authors: Mikhail Dubrovin, TJ Lane, Christopher O'Grady
+"""
+
+
+class Projector(object):
```

Comment:
@licode @tacaswell Hi guys, following Li's advice I have left the code in accumulators.  Following Tom's advice I have renamed the classes BinImage and RadialBinImage.  Let me know if you have further thoughts.  Thanks!